docs/themes/godocs-2/layouts/partials/default.html

```diff
@@ -6,11 +6,10 @@
         <ul class="sidenav pt-4 pe-4 border-end border-default">
           {{ $currentNode := . }}
           {{range .Site.Home.Sections.ByWeight}}
-          <!-- not render any single page, like contact page. -->
+          <!-- render sections with one or more pages. -->

```

Comment:
I had to make this change in order for the FAQ page to be displayed on the sidebar.  Otherwise, I would have needed to add a blank subsection which is not necessary. This did not impact the contact page, which only gets rendered if the button is clicked.